dts/bindings/sensor/nxp,fxos8700-spi.yaml

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2018, NXP
+# SPDX-License-Identifier: Apache-2.0
+
+description: FXOS8700 6-axis accelerometer/magnetometer sensor
+
+compatible: "nxp,fxos8700"
+
+include: ["spi-device.yaml", "nxp,fxos8700-common.yaml"]
```

Comment:
where are the changes adding support for SPI? If added later this breaks bisectability.

Reply:
SPI changes have been added to the fxos8700 sensor driver, please -  drivers/sensor/fxos8700.c